beacon_node/http_api/src/lib.rs

```diff
@@ -2356,6 +2356,41 @@ pub fn serve<T: BeaconChainTypes>(
             })
         });
 
+    // GET lighthouse/seen_validators?ids,epochs
+    let get_lighthouse_seen_validators = warp::path("lighthouse")
+        .and(warp::path("seen_validators"))
+        .and(warp::path::end())
+        .and(warp::query::<api_types::SeenValidatorQuery>())
+        .and(chain_filter.clone())
+        .and_then(
+            |query: api_types::SeenValidatorQuery, chain: Arc<BeaconChain<T>>| {
+                blocking_json_task(move || {
+                    let indices: Vec<usize> = chain
+                        .head_beacon_state()
+                        .map_err(warp_utils::reject::beacon_chain_error)?
+                        .validators
+                        .iter()
+                        .enumerate()
+                        // filter by validator id(s) if provided
+                        .filter_map(|(index, validator)| {
+                            query
+                                .ids
+                                .0
+                                .iter()
+                                .find(|pubkey| *pubkey == &validator.pubkey)
+                                .map(|_| index)
+                        })
+                        .collect();
+
+                    let seen_validator =
```

Comment:
Maybe it would be nice to return a list of the offending validator indices to make it easier for the user to determine which validator is duplicated.